cmd/tuf/app/init.go

```diff
@@ -258,7 +261,7 @@ func InitCmd(ctx context.Context, directory, previous string, threshold int, tar
 	if err != nil {
 		return err
 	}
-	if err := setMetaWithSigKeyIDs(store, "targets.json", t, allRootKeys); err != nil {
+	if err := setMetaWithSigKeyIDs(store, "targets.json", t, keys); err != nil {
```

Comment:
It's not clear to me why this is change needed. Are `keys` always the same as `allRootKeys`?

Reply:
https://github.com/sigstore/root-signing/blob/b96c980798d145d1c0c4edbc0d2818dc7820bb0d/cmd/tuf/app/init.go#L168-L171

Reply:
Thanks!